aries-site/src/examples/templates/table-customization/components/TableCustomizationExample.js

```diff
@@ -126,109 +97,59 @@ export const TableCustomizationExample = () => {
-                    />
-                  </Box>
-                ) : (
-                  <StyledButton
-                    id="search-button"
-                    icon={<Search />}
-                    onClick={() => setSearchFocused(true)}
-                  />
-                )}
-                {(size !== 'small' || !searchFocused) && (
-                  <Filters
-                    data={allData}
-                    filtering={filtering}
-                    setFiltering={setFiltering}
-                    setData={setData}
-                    filters={filters}
-                    setFilters={setFilters}
-                    filterData={filterData}
-                    setSearch={setSearch}
-                  />
-                )}
-              </Box>
-              <RecordSummary
+      <FiltersProvider>
+        <Header pad={{ top: 'medium' }}>
+          <Box gap="xsmall" fill="horizontal">
+            <Heading level={2} margin="none">
+              Users
+            </Heading>
+            <Box direction="row" justify="between">
+              <FilterControls
                 data={allData}
-                filteredData={data}
-                filtering={filtering}
+                filters={filtersConfig}
+                searchFilter={{ placeholder: 'Search users...' }}
               />
-            </Box>
-            <Box direction="row" height="fit-content">
-              <DropButton
-                icon={<Edit />}
-                dropAlign={{ top: 'bottom', right: 'right' }}
-                dropContent={
-                  <ColumnSettings
-                    columns={COLUMNS}
-                    visibleColumns={visibleColumns}
-                    setVisibleColumns={setVisibleColumns}
-                  />
-                }
-              />
-              <Menu label="Actions" items={[]} />
+              <Box direction="row" height="fit-content">
+                <DropButton
+                  icon={<Edit />}
+                  dropAlign={{ top: 'bottom', right: 'right' }}
+                  dropContent={
+                    <ColumnSettings
+                      columns={COLUMNS}
+                      visibleColumns={visibleColumns}
+                      setVisibleColumns={setVisibleColumns}
+                    />
+                  }
+                />
+                <Menu label="Actions" items={[]} />
+              </Box>
             </Box>
           </Box>
-        </Box>
-      </Header>
-      <Results data={data} columns={visibleColumns} />
+        </Header>
+        <Results columns={visibleColumns} />
+      </FiltersProvider>
     </Box>
   );
 };
 
-const Results = ({ data, columns }) => {
+const Results = ({ columns }) => {
   const [select, setSelect] = useState([]);
+  const { filteredResults } = useFilters();
 
   return (
-    <Box fill="vertical" overflow="auto">
+    <Box fill overflow="auto">
       <DataTable
-        data={data}
+        data={filteredResults}
```

Comment:
In fullscreen, dark mode:

![Screen Shot 2021-06-03 at 11 22 04 AM](https://user-images.githubusercontent.com/1756948/120686223-fff23c80-c45d-11eb-8982-ad0aabe3e2ca.png)

We could do:

```suggestion
        background="background"
        data={filteredResults}
```

Or:

```suggestion
        background="background-front"
        data={filteredResults}
```
And add `background="background-front"`on Line 94-ish.

Reply:
why the contrasting color for the table? That looks odd and distracting to me, especially with the checkbox column background being different

Reply:
oh or you are arguing to change the example background to `background` so it has contrast in dark mode but then you also have to change the DataTable since the general background bleeds through into that?

Reply:
The screenshot is how it currently renders when in fullscreen dark mode (which doesn't look correct).

Trying to address so that it looks correct. I don't have a preference towards either approach, just as long as the background is uniform across the entire table.

Reply:
> but then you also have to change the DataTable since the general background bleeds through into that?

Right. We can either just apply "background" to the DataTable and all looks good. Or we could apply "background-front" to the DataTable, but I believe we would also need to apply "background-front" to the example Box as well to make things look correct.

I am good with either approach.

Reply:
The whole thing is lighter color for me. Not sure why the difference (safari?) but I'll do the above.